docs/guides/latest/hooks/backup-and-restore-hooks.md

```diff
@@ -0,0 +1,720 @@
+---
+title: Backup & Restore Hooks | Stash
+menu:
+  docs_{{ .version }}:
+    identifier: backup-and-restore-hooks
+    name: Backup & Restore Hooks
+    parent: hooks
+    weight: 20
+product_name: stash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Backup & Restore Hooks
```

Comment:
Should we write, "This is an advacned concept. Try simpler things first."

Reply:
This doc already comes at the bottom of the sidebar. Isn't it natural to try the upper docs first? Btw, I agree to include.